test-tools/test_state_recover.sh

```diff
@@ -177,38 +184,68 @@ function step2_delete_old_state()
     echo "Created signed state amount: $signed_state_amount"
 
     #random delete some states
-    random_num=$(( ( RANDOM % ($signed_state_amount - 1) + 1 ) ))
+    state_delete_num=$(( ( RANDOM % ($signed_state_amount - 1) + 1 ) ))
 
-    echo "Deleting $random_num signed states"
+    echo "Deleting $state_delete_num signed states"
 
     # remove most of states only leaving one
     for ((i=1;i<=random_num;i++)); do
```

Comment:
Did we change the rand_num to state_delete_num already?

Reply:
yes, state_delete_num is a global variable now

Reply:
Then should the random_num be changed to state_delete_num on line 192 (old linen number 185)?